docs/shared/monorepo-ci-github-actions.md

```diff
@@ -31,6 +31,9 @@ jobs:
 ```
 
 The `pr` and `main` jobs implement the CI workflow. Setting `timeout-minutes` is needed only if you have very slow tasks.
+> **Note**
```

Comment:
can you please add a leading "blank line" here?

Reply:
Yeah!